src/gasPriceOracle/adapters/arbitrum.ts

```diff
@@ -10,6 +10,10 @@ import { GasPriceEstimateOptions } from "../oracle";
  * Reference: https://docs.arbitrum.io/how-arbitrum-works/gas-fees so we hardcode the priority fee
  * to 1 wei.
  * @param provider Ethers Provider
+ * @param {GasPriceEstimateOptions} opts See notes below on specific parameters.
```

Comment:
We shouldn't need to type params in TSDoc formatting
```suggestion
 * @param opts See notes below on specific parameters.
```